package.json

```diff
@@ -15,7 +15,7 @@
         "details": "webpack --config webpack.dev.js --mode development --stats-error-details",
         "analyze": "webpack --config webpack.dev.js --mode production --json > stats.json",
         "prepublish": "yarn build",
-        "test": "jest",
+        "test": "jest --collectCoverage=true",
```

Comment:
This may significantly slow down tests.

Reply:
This was already enabled via the previously true config version on line 245. Adding it here and disabling it in the below config results in the coverage still being reported when running the entire test suite via `yarn run test`, but no output when running a single test within vscode or command line running jest directly.